packages/angular/cli/src/commands/cache/clean/cli.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { promises as fs } from 'fs';
+import { Argv } from 'yargs';
+import {
+  CommandModule,
+  CommandModuleImplementation,
+  CommandScope,
+} from '../../../command-builder/command-module';
+import { getCacheConfig } from '../utilities';
+
+export class CacheCleanModule extends CommandModule implements CommandModuleImplementation {
+  command = 'clean';
+  describe = 'Deletes cache from disk.';
```

Comment:
Nit: "cache" is kind of vague, maybe "build cache" or "disk cache" would more clearly articulate what we mean here? I'm mainly concerned that devs might misinterpret this as the NPM cache in `node_modules/`.

If you do change this, don't forget to also update the other descriptions to use consistent language.

Reply:
Agreed.